src/components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistoryCurrentSearch.js

```diff
@@ -7,22 +7,41 @@ import SubscriberHistorySentCampaigns from './SubscriberHistorySentCampaigns/Sub
 const SubscriberHistoryCurrentSearch = ({ searchText, dependencies: { dopplerApiClient } }) => {
   const [state, setState] = useState({ loading: true });
 
-  const showSubscriberCampaigns = () =>
-    setState((prevState) => ({ ...prevState, showSubscriberCampaigns: true }));
+  const showSubscriberCampaigns = (subscriber) =>
+    setState((prevState) => ({
+      ...prevState,
+      showSubscriberCampaigns: true,
+      selectedSubscriber: subscriber,
+    }));
 
   useEffect(() => {
     const fetchData = async () => {
       setState({ loading: true });
-      const response = await dopplerApiClient.getSubscriber(searchText);
+      const response = await dopplerApiClient.getSubscribers(searchText);
       if (response.success) {
-        const subscriber = {
-          ...response.value,
-          firstName: response.value.fields.find((x) => x.name === 'FIRSTNAME'),
-          lastName: response.value.fields.find((x) => x.name === 'LASTNAME'),
-        };
-        setState({ loading: false, subscriber: subscriber });
+        const subscribers = response.value.items.map((subscriber) => {
+          return {
+            ...subscriber,
+            firstName: subscriber.fields.find((x) => x.name === 'FIRSTNAME'),
+            lastName: subscriber.fields.find((x) => x.name === 'LASTNAME'),
+          };
+        });
+        setState({ loading: false, subscribers: subscribers });
       } else {
-        setState({ loading: false });
+        // Fallback for production
```

Comment:
Beware of this fallback, if the endpoint does not exist in production you will get a 404, not a success false. 
![image](https://user-images.githubusercontent.com/2439363/71582205-d3f3be80-2ae7-11ea-8496-3baceeea0c55.png)

Reply:
I would add a try in `getSubscribers` call and as a catch a fallback to `getSubscriber`.

Reply:
ok i added. now im waiting for build to test in cdn

Reply:
We still have a problem... 
![image](https://user-images.githubusercontent.com/2439363/71598402-636b9280-2b25-11ea-9f57-9267c4839368.png)